packages/nx-plugin/package.json

```diff
@@ -49,7 +49,8 @@
   },
   "dependencies": {
     "@nrwl/devkit": "^15.3.0",
-    "@nrwl/linter": "^15.3.0"
+    "@nrwl/linter": "^15.3.0",
+    "typescript": "^4.9.4"
```

Comment:
This might be a bit too greedy to make the users download this specific version of `typescript`.
Even more that most likely no one will ever the types of the package since it will be used by `nx`.

Isn't there a way to get around with `devDevpendencies` and/or `peerDependencies`?

Reply:
done